lib/app-service.js

```diff
@@ -31,7 +40,20 @@ function AppService(config) {
                  *   console.log(logLine);
                  * });
                  */
-                self.emit("http-log", str);
+                var sep = str.match(ACCESS_TOKEN_RE);
+                sep.shift(); // Remove entire match as first element
+
+                var redacted_str = str;
```

Comment:
Please use `camelCase` rather than underscores. Code styles in general will vary based on the project: in Synapse you use underscores instead. The general hard-and-fast rule is to follow the style that is around you, and that is `camelCase`.